configuration.toml

```diff
@@ -0,0 +1,14 @@
+[server]
```

Comment:
I'm not sure about this file, I think is a better experience to avoid manual steps to run the tool. Maybe we can stick to default values in the code and force the required input. (also binary requires this file too and doesn't feel good)

However, the CI will override and not persist the changes.